.github/workflows/conda-env-create-and-pytest.yml

```diff
@@ -0,0 +1,70 @@
+name: conda-env-create-and-pytest
```

Comment:
```suggestion
name: catalog-build-pytests-in-conda-env
```

Reply:
the workflow file could also be renamed to catalog-build-pytests-in-conda-env? (or anything that is a bit descriptive as this is an important dynamic catalog building test)

Reply:
i'm OK with finding a new name, but let's avoid "build" in this context. build is very overloaded here. we have catalogbuilder building catalogs, a pipeline building an environment, a pipeline building a package using conda build... 

ideally, some day, this dynamic catalog creation test exists within a pytest suite

Reply:
how about `conda-env-create-run-tests`?